### DISCUSSION TITLE
Autioindex

### DISCUSSION THREAD

**Discussion by aptmess** [milvus-io/milvus#30560] [Q&A and General discussion] [answered]
Good day!

How to understand (milvus-2.3.7) what is autoindex? What are the parameters of this index?

![image](https://github.com/milvus-io/milvus/assets/43338025/864347e1-86db-4b5c-851c-f847e2444cec)

Thanks in advance!

**Comment by yhmo** [comment]
AUTOINDEX is only available for Zilliz cloud. The background index could be a graph index or some other indexes. The detail is invisible to users.

**Comment by aptmess** [comment, marked as answer]
Also, the information is written in file `configs/milvus.yaml`:

```
autoIndex:
  params:
    build: '{"M": 18,"efConstruction": 240,"index_type": "HNSW", "metric_type": "IP"}'
```

So, this is default index for every collection, for those who was answering.

**Reply by yhmo** [reply]
This is the default index for Milvus community edition.
On Zilliz cloud, autoindex could be different.

**Reply by aptmess** [reply]
I was asking about default community because I don't use cloud, thanks! However, the information about cloud index doesn't look significant, isn't it? You answered only that it 'could be different' and it doesn't provide any info and technical details, so no offense... 

Thanks for answering!
